app/Services/Adselect.php

```diff
@@ -95,21 +93,21 @@ public function getBanners(array $requests)
 //             return [];
 //         }
 
-        $responses = [];
-
-        foreach ($requests as $request) {
-            $responses[] = [
-                'request_id' => $request['request_id'],
-                'banner_id' => $request['request_id'] + 1,
-            ];
-        }
-
-        return $responses;
+//        $responses = [];
+//
+//        foreach ($requests as $request) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)